src/cli/global/edit.rs

```diff
@@ -0,0 +1,44 @@
+use crate::global::Project;
+use clap::Parser;
+use miette::IntoDiagnostic;
+use pixi_config::{Config, ConfigCli};
+
+/// Edit the global manifest file
+///
+/// Opens your default editor to edit the global manifest file.
+#[derive(Parser, Debug)]
+#[clap(verbatim_doc_comment)]
+pub struct Args {
+    /// Answer yes to all questions.
+    #[clap(short = 'y', long = "yes", long = "assume-yes")]
+    assume_yes: bool,
+
+    #[clap(flatten)]
+    config: ConfigCli,
+}
+
+pub async fn execute(args: Args) -> miette::Result<()> {
+    let config = Config::with_cli_config(&args.config);
+    let project = Project::discover_or_create(args.assume_yes)
+        .await?
+        .with_cli_config(config.clone());
```

Comment:
```suggestion
    let project = Project::discover_or_create(args.assume_yes)
        .await?
        .with_cli_config(config.clone());
```

Just tried adding something invalid with `pixi global edit` and then running `pixi global edit` again. This line makes this fail.

Reply:
hmm good point. Lets assume the file already exists. And otherwise just create it.

Reply:
Changed that.